_api-reference/ingest-apis/index.md

```diff
@@ -9,6 +9,160 @@ redirect_from:
 
 # Ingest APIs
 
```

Comment:
@kolchfa-aws Please provide a peer review of this PR, if possible by end of week. Reach out if we need to sync on it. Thank you!